### PR TITLE
Update cookie contents

### PR DESCRIPTION
**Pull request by lilyminium**
Minor changes here:
* added more structure to the `data/` directory:
    * `files.py` thing to mimic `MDAnalysisTests.datafiles`
    * Replaced original template data with logo

I was tossing up having the options ask if mdakits wanted to include new analysis, new converters, topology parsers, etc (https://github.com/MDAnalysis/cookiecutter-mda/pull/2). Then we could add new module templates, e.g. `module/analysis/analysis_template.py` that starts a new AnalysisBase class . Would that be helpful, or inconvenient clutter?

### CODE REVIEW

**Review by IAlibay — APPROVED** [review]

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
-"""{{cookiecutter.description}}"""
+"""
+{{cookiecutter.project_name}}
+{{cookiecutter.description}}
+"""
 
 # Add imports here
 from .{{cookiecutter.first_module_name}} import *
```
Comment (orbeckst) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py:7]: Do we really want to encourage `*` imports?

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/data/files.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+"""
+
+__all__ = [
+    "MDANALYSIS_LOGO",  # example file of MDAnalysis logo
+]
+
+from pkg_resources import resource_filename
+
+MDANALYSIS_LOGO = resource_filename(__name__, "mda.txt")
```
Comment (orbeckst) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/data/files.py:17]: cute!

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+"""
+Global pytest fixtures
+"""
```
Comment (orbeckst) [inline {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/tests/conftest.py:3]: maybe include a comment/link to the pytest docs on global pytest fixtures?

**Review by orbeckst — COMMENTED** [review]
Looks quite useful already.

Perhaps include an example test that checks that the MDANALYSIS_LOGO file exists — showing _how_ to access the data files in a test would be a useful pattern and seed best practices.

**Comment by orbeckst** [comment]
A new analysis template would be quite good because it already covers typical scenarios.

If you think you can do the same for converters, coordinate & topology I/O then even better. But I think that could also come later and just raising an issue would be sufficient.

**Comment by IAlibay** [comment]
I think generally the idea of having templates for different things was viewed positively last time we discuss it, so we should really be going in that direction. That being said I'm happy having that be a "thing we do in the near future PRs"

File: {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py (hunk 1 above)
Comment (lilyminium) [reply {{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py:7]: No, I changed it to `canvas`.

**Comment by lilyminium** [comment]
Added a fixture and test:

`conftest.py` (https://github.com/lilyminium/test_mda_pip_rtd/blob/main/test_mda_pip_rtd/tests/conftest.py):
```
from test_mda_pip_rtd.data.files import MDANALYSIS_LOGO


@pytest.fixture
def mdanalysis_logo_text() -> str:
    """Example fixture demonstrating how data files can be accessed"""
    with open(MDANALYSIS_LOGO, "r", encoding="utf8") as f:
        logo_text = f.read()
    return logo_text
```

`test.py` (https://github.com/lilyminium/test_mda_pip_rtd/blob/main/test_mda_pip_rtd/tests/test_test_mda_pip_rtd.py):

```
def test_mdanalysis_logo_length(mdanalysis_logo_text):
    """Example test using a fixture defined in conftest.py"""
    logo_lines = mdanalysis_logo_text.split("\n")
    assert len(logo_lines) == 46, "Logo file does not have 46 lines!"
```

**Comment by lilyminium** [comment]
I also raised https://github.com/MDAnalysis/cookiecutter-mda/issues/11 as a center for discussing adding new templates.

**Review by orbeckst — APPROVED** [review]
Very nice skeleton for tests. I would merge it as it is now.

Adding additional templates should follow #11 .

(As a thought for _future_ test examples, should we encourage using the numpy testing assertions that we use in MDAnalysis excessively or should we just let everyone discover them for themselves?)

**Comment by lilyminium** [comment]
@orbeckst yes I think that's a good idea! Some nice-to-have demos would include:
* assert_allclose, assert_equal
* pytest.parametrize
* pytest.skipif / pytest.xfail

but that can get added later :)

**Pull request merged**